openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -44,7 +44,7 @@ This specification defines an API for the issuance of Verifiable Credentials.
 
 # Introduction
 
-This specification defines an API that is used to issue Verifiable Credentials. W3C formats [@VC_DATA] as well as other Credential formats, like [@ISO.18013-5], are supported. 
+This specification defines an OAuth protected API for the issuance of Verifiable Credentials. Credentials can be of any format, including, but not limited to, W3C formats [@VC_DATA], mDL [@ISO.18013-5], and SD-JWT VC [@I-D.ietf-oauth-sd-jwt-vc].
```

Comment:
I don't want to be picky but not all endpoints defined in this specification are OAuth protected.

You might also want to use "Verifiable Credentials" in the second sentence.

Reply:
we are not talking about Authz/Token endpoints here. All endpoints used for the issuance (Credential, Batch, Deferred, Notification) are OAuth protected.

Reply:
@Sakurann , I didn't refer to Authz/Token endpoints. This specification also defines & introduces /.well-known/openid-credential-issuer that isn't an OAuth protected API.

Reply:
Let's not be picky at that level in the introduction.